clusterman/util.py

```diff
@@ -64,6 +64,14 @@ class ClustermanResources(NamedTuple):
     gpus: float = 0
 
 
+class ClusterNotFoundError(Exception):
```

Comment:
Can we stick this in `clusterman/exceptions.py`?

Reply:
Yep.